admin/anspress-admin.php

```diff
@@ -720,7 +720,8 @@ private static function check_pages_exists() {
 
 			$pages_in = array();
 			foreach ( $pages_slug as $slug ) {
-				$pages_in[] = $opt[ $slug ];
+				$pages      = isset( $opt[ $slug ] ) ? $opt[ $slug ] : '';
```

Comment:
@mebishalnapit I think it would be better to add if condition instead of appending empty array item.

Reply:
@rahularyan Updated as requested.